models/NeatlineExhibit.php

```diff
@@ -82,8 +106,15 @@ public function getNumberOfRecords()
     public function getRecordUrl($action = 'show')
     {
         $urlHelper = new Omeka_View_Helper_Url;
-        $params = array('action' => $action, 'id' => $this->id);
-        return $urlHelper->url($params, 'neatlineActionId');
+        $checkUrl = $urlHelper->url();
+        if (strpos($checkUrl, '/admin/') !== false) {
```

Comment:
This works well, but seems a tad brittle, as a public URL might contain that substring, hypothetically. Out of curiosity: are there examples elsewhere in Omeka-land, or Plugin-land, of handling admin vs public user actions? Not a blocker.